mlflow/metrics/__init__.py

```diff
@@ -191,6 +200,126 @@
 .. _rougeLsum: https://huggingface.co/spaces/evaluate-metric/rouge
 """
 
+# General Regression Metrics
+
+mae = make_metric(
+    eval_fn=_mae_eval_fn,
+    greater_is_better=False,
+    name="mae",
```

Comment:
We don't want to change the names from how they are logged right now in mlflow.evaluate, look in `_get_regressor_metrics` in `default_evaluator.py` to see how they are named rn (e.x. mean_absolute_error for this specific one).

Reply:
okay I'll keep the full names